R/pool.R

```diff
@@ -187,69 +280,47 @@ rubin_rules <- function(ests, ses, v_com) {
     return(ret_obj)
 }
 
-#' @importFrom stats qt pt var
-#' @rdname pool
-#' @export
-pool_.rubin <- function(results, conf.level, alternative, type) {
-    ests <- results$est
-    ses <- results$se
-    dfs <- results$df
-    alpha <- 1 - conf.level
-
-    v_com <- unique(dfs)
-
-    assert_that(
-        all(!is.na(ses)),
-        msg = "Standard Errors for Rubin's rules can not be NA"
-    )
-
-    assert_that(
-        length(v_com) == 1,
-        msg = "Degrees of freedom should be consistent across all samples"
-    )
 
-    res_rubin <- rubin_rules(
-        ests = ests,
-        ses = ses,
-        v_com = v_com
-    )
-
-    ret <- normal_ci(
-        point = res_rubin$est_point,
-        se = sqrt(res_rubin$var_t),
-        alpha = alpha,
-        alternative = alternative,
-        qfun = qt,
-        pfun = pt,
-        df = res_rubin$df
-    )
 
-    return(ret)
-}
 
 
 
 
-#' Title
+#' Bootstrap Pooling via Percentiles
 #' @description
-#' TODO
-#' @param est TODO
-#' @param conf.level TODO
-#' @param alternative TODO
+#' Creates a point estimate, confidence interval and p-values using
+#' percentiles. Note that quantile "type=6" is used,
+#' see [stats::quantile()] for details.
+#' @param est a numeric vector of point estimates from each bootstrap sample
+#' @inheritParams pool
+#' @details
+#' The point estimate is taken to be the first element of est. The remaining
```

Comment:
for example, here "est" should not be "`est`"?

Reply:
done